client/question/domain/FeedbackParagraphObjectFactory.js

```diff
@@ -19,8 +19,9 @@
 
 tie.factory('FeedbackParagraphObjectFactory', [
   'PARAGRAPH_TYPE_TEXT', 'PARAGRAPH_TYPE_CODE', 'PARAGRAPH_TYPE_SYNTAX_ERROR',
+  'PARAGRAPH_TYPE_OUTPUT',
```

Comment:
Have line breaks here be the same as the ones in the function signature below -- this makes it easier to visually check that the dependencies line up.

Reply:
done